source/User_Guide/Settings/billing.md

```diff
@@ -1,63 +1,127 @@
+
+**Billing Frozen**
+
+If you find your account suddenly unable to send mail within the first week of the month, your account could be in a billing frozen state. This state occurs when the card on file for your account gets declined when we attempted to process payment on the first of the month.
+
+To reactivate your account, please update your credit card on file by logging into the SendGrid UI and going to the [Plan & Billing Details page](https://app.sendgrid.com/settings/billing). You can then change or re-enter the card information and then click **Save**. Once you update the card on file, the payment will process automatically, and the account is unfrozen.
 
 {% anchor h2 %}
-Changing Your Plan
+Email credits
 {% endanchor %}
 
-To change your plan, click the “Change Plan” button next to your billing details. You will be shown the plan options that are available to you, whether you are upgrading or downgrading your SendGrid service. Simply click the plan you would like and follow the prompts.
+Every email processed by SendGrid uses an email credit. Free, Essentials, Pro, and Premier accounts will see their email credits reset the 1st day of each month at 12:01 PST.
 
 {% anchor h2 %}
-Cancel Account
+Overages
 {% endanchor %}
 
+Depending on your package, you will be charged a small amount per-email for each email that you send beyond the credit limit of your current package. Any overage charges get added to the invoice of the month _after_ the over-limit sending occurred.
+
 {% info %}
-Canceled accounts are not deleted. Accounts are changed to our [free]({{site.site_url}}/free?mc=SendGrid%20Documentation) plan.
+Upgrading your account does NOT absorb overage charges already incurred, so make sure you are using the right package for your usage case.
 {% endinfo %}
 
+{% anchor h2 %}
+Requesting a refund
+{% endanchor %}
+
+You must meet both criteria below for a refund to be considered:
+
+- You have not used your account in the current calendar month.
+- You have no overages, and you have no contacts stored in Marketing Campaigns.
+
+SendGrid packages operate off of a monthly subscription model, not direct usage. We will not issue a refund if you have used your account in the same calendar month as your request. Instead please cancel your account to avoid any future subscription charges.
+
+Refunds are only applicable to the subscription charge on your account, not for overage or contact storage charges from the prior month. We hold our customers responsible for managing their account credit limits and contact storage.
+
+If you meet these conditions, please respond directly to the invoice email you have received, or email billing@sendgrid.com to begin the discussion about a possible refund.
+
+{% anchor h2 %}
+Changing your plan
+{% endanchor %}
+
+To change your plan, click the “Change Plan” button next to your [billing details](https://app.sendgrid.com/settings/billing). Click the plan you would like and follow the prompts.
```

Comment:
To change your plan, click **Change Plan** next to your [billing details](https://app.sendgrid.com/settings/billing). Click the plan you would like and follow the prompts.